it/test/uk/gov/hmrc/alcoholdutyreturns/controllers/ReturnsIntegrationSpec.scala

```diff
@@ -59,6 +59,23 @@ class ReturnsIntegrationSpec extends ISpecBase {
         verifyPost(calculateDutyDueByTaxTypeUrl)
         verifyPost(submitReturnUrl)
       }
+
+      "return 400 BAD_REQUEST and the submission created response when successful" in new SetUp {
+        stubAuthorised()
+        stubPost(calculateDutyDueByTaxTypeUrl, OK, Json.toJson(calculateDutyDueByTaxTypeRequestForExampleSubmission).toString(), Json.toJson(calculatedDutyDueByTaxTypeForExampleSubmission).toString())
+        stubPost(submitReturnUrl, CREATED, Json.toJson(returnSubmission).toString(), Json.toJson(returnCreatedSuccess).toString())
```

Comment:
Does this need to be stubbed as it is never called?

Reply:
It should be never called, though I do verify that's the case below.